### PR TITLE
MNT Remove TODO comments

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/.github/issues/116

### CODE REVIEW

File: src/File.php [hunk 1]
```diff
@@ -1301,7 +1299,6 @@ public function getTag()
      */
     public function BackLinkTracking()
     {
-        // @todo - Implement PolymorphicManyManyList to replace this
```
Comment (sabina-talipova) [inline src/File.php:1304]: Open ticket: https://github.com/silverstripe/silverstripe-assets/issues/575

File: src/File.php [hunk 2]
```diff
@@ -711,8 +711,6 @@ protected function onBeforeWrite()
                 // This logic is also used in the Folder subclass, but we're querying
                 // for duplicates on the File base class here (including the Folder subclass).
 
-                // TODO Add read lock to avoid other processes creating files with the same name
-                // before this process has a chance to persist in the database.
```
Comment (sabina-talipova) [inline src/File.php:715]: Open ticket: https://github.com/silverstripe/silverstripe-assets/issues/578

File: src/Upload.php [hunk 3]
```diff
@@ -27,7 +27,6 @@
  * (see http://doc.silverstripe.org/secure-development#filesystem).
  * {@link File::$allowed_extensions} provides a good start for a list of "safe" extensions.
  *
- * @todo Allow for non-database uploads
```
Comment (sabina-talipova) [inline src/Upload.php:30]: Open ticket: https://github.com/silverstripe/silverstripe-assets/issues/576

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**